encoder_test.go

```diff
@@ -5,7 +5,10 @@ import (
 	"testing"
 	"time"
 
+	"go.uber.org/zap/buffer"
```

Comment:
This should be grouped with the others.

Reply:
Just saw this. IIRC (from 8 months ago) I believe this was the output from gofmt.

Reply:
Yea, `gofmt` will preserve the grouping of imports. If you can just throw it into the same group as the other imports then I'll merge this.

Reply:
Done!